src/main/telemetry/smartport.c

```diff
@@ -812,7 +812,7 @@ static bool serialCheckQueueEmpty(void)
 
 void handleSmartPortTelemetry(void)
 {
-    const uint32_t requestTimeout = millis() + SMARTPORT_SERVICE_TIMEOUT_MS;
+    const uint32_t requestTimeout = micros() + SMARTPORT_SERVICE_TIMEOUT_MS * 1000;
```

Comment:
(maybe refactor to 'SMARTPORT_SERVICE_TIMEOUT_US')

Reply:
Yes, I am also for using `#define SMARTPORT_SERVICE_TIMEOUT_US 1000` here.